docs/source/x_ext.rst

```diff
@@ -366,24 +403,24 @@ Compressed interface
   :name: Compressed request type
   :class: no-scrollbar-table
 
-  +------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------+
-  | **Signal**             | **Type**                | **Description**                                                                                                 |
-  +------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------+
-  | ``instr``              | logic [15:0]            | Offloaded compressed instruction.                                                                               |
-  +------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------+
-  | ``id``                 | :ref:`id_t <id>`        | Identification number of the offloaded compressed instruction.                                                  |
-  +------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------+
+  +------------------------+--------------------------+-----------------------------------------------------------------------------------------------------------------+
+  | **Signal**             | **Type**                 | **Description**                                                                                                 |
+  +------------------------+--------------------------+-----------------------------------------------------------------------------------------------------------------+
+  | ``instr``              | logic [15:0]             | Offloaded compressed instruction.                                                                               |
+  +------------------------+--------------------------+-----------------------------------------------------------------------------------------------------------------+
+  | ``hartid``             | :ref:`hartid_t <hartid>` | Identification of the hart offloading the instruction.                                                          |
```

Comment:
Why is the 'id' no longer used?

Reply:
it has no purpose. its only use was to change from instr on the fly without deasserting valid. now hartid has that function

Reply:
This conflicts with the section 'Identification' where an id can get issued on the compressed interface and that same id would then get used by the issue interface. I don't know if that is important or not, but conceptually I think it is nice if all interfaces use the same id if they talk about the same offloaded instruction.

Reply:
The compressed interface is special, in that it does not offload anything or work on an offloaded instruction. Its just attaching a combinatorial logic block. As such, it does not need to handle IDs. Thus, I find it quite logical that it wouldn't have the ID. I did update in ebf3a7e to have a consistent proposal.